packages/node/package.json

```diff
@@ -85,6 +85,7 @@
     "@opentelemetry/instrumentation-nestjs-core": "0.39.0",
     "@opentelemetry/instrumentation-pg": "0.43.0",
     "@opentelemetry/instrumentation-redis-4": "0.41.0",
+    "@opentelemetry/instrumentation-tedious": "0.12.0",
```

Comment:
`0.12.0` supports tedious `>=1.11.0 <18`. See [here](https://www.npmjs.com/package/@opentelemetry/instrumentation-tedious/v/0.12.0). 
`0.13.0` supports tedious `>=1.11.0 <20`. See [here](https://www.npmjs.com/package/@opentelemetry/instrumentation-tedious/v/0.13.0) - released just yesterday!

Currently using `tedious ^18.6.1` with `instrumentation-tedious 0.12.0`. That's why no spans are emitted and the integration tests are failing. 

Change to `0.13.0`.